src/datalogger.cpp

```diff
@@ -1005,7 +1020,7 @@ void Datalogger::powerUpSwitchableComponents()
 
   // turn on 5v booster for exADC reference voltage, needs the delay
   // might be possible to turn off after exADC discovered, not certain.
-  gpioPinOn(GPIO_PIN_4);
+  gpioPinOn(GPIO_PIN_3);
   
```

Comment:
Need to get this out of here and into the driver.

Reply:
Don't have a driver for the booster / actuator yet

Reply:
oh that's interesting, are you thinking of the booster having it's own driver? i was thinking that since the methane sensor needs that, it would go into the methane sensor driver, which could be assigned two ports to use as well.